js/utils.js

```diff
@@ -17,6 +17,55 @@ Fliplet.Registry.set('dynamicListUtils', (function () {
   };
   var computedFieldClashes = [];
 
+  function setLineClamps () {
+    setClampHeight()
+      .then(function (elements) {
+        Superclamp.register(elements);
+        return Promise.resolve(elements);
+      })
+      .then(function (elements) {
+        removeClampHeight(elements);
+      })
+      .catch(function (selector) {
+        console.error('Fail to clamp line in the selector ' + selector);
+      });
+  }
+
+  function setClampHeight (selector) {
+    selector = selector || '[data-line-clamp]';
+
+    return new Promise(function (resolve, reject) {
+      var elements = document.querySelectorAll(selector);
+      
+      if (!elements.length) {
+        reject(selector);
+      }
+
+      for (var i = 0; i < elements.length; i++) {
+        var element = elements[i];
+        var lineHeightValue = window.getComputedStyle(element)
+          .getPropertyValue('line-height').match(/[a-zA-Z]+|[0-9]+(?:\.[0-9]+|)/g);
+        var lineHeightMeasure  = lineHeightValue[1];
+        var elementHeight = parseFloat(lineHeightValue[0]);
+        var clampLines = element.dataset.lineClamp;
+        var elementHeight = elementHeight * clampLines;
+
+        element.style.height = elementHeight + lineHeightMeasure;
+      }
+
+      resolve(elements);
+    });
+  }
+
+  // To avoid large distance between block we remove height at the element
+  function removeClampHeight (elements) {
+    for (var i = 0; i < elements.length; i++) {
+      var element = elements[i];
+
+      element.removeAttribute("style");
```

Comment:
What if the element contains other styles? It should only be the height that gets removed.